.github/workflows/nvmx.yml

```
@@ -0,0 +1,19 @@
+name: NVMx testing
+on: push
```

Do you want to run it on push to every branch. What about something like

```suggestion
on: 
  push:
    branches:
      - cas-534-nvmx
```

---

Can't hurt to filter somewhat :+1: